docs/source/change_log.md

```diff
@@ -1,3 +1,25 @@
+# v 0.8.0 (2/12/2022)
+
+## Highlights
+- Add BEVFormer and improve the performance of BEVFormer ([#224](https://github.com/alibaba/EasyCV/pull/224))
+- Add DINO++ and support objects365 pretrain ([#242](https://github.com/alibaba/EasyCV/pull/242))
+
+## New Features
+- Add DeiT of Hydra Attention version ([#220](https://github.com/alibaba/EasyCV/pull/220))
+- Add EdgeViTv3 ([#214](https://github.com/alibaba/EasyCV/pull/214))
+- Add BEVFormer and improve the performance of BEVFormer ([#224](https://github.com/alibaba/EasyCV/pull/224))
+- Add DINO++ and support objects365 pretrain ([#242](https://github.com/alibaba/EasyCV/pull/242))
+
+## Improvements
+- Unify the parsing method of config scripts, and support both local and pai platform products ([#235](https://github.com/alibaba/EasyCV/pull/235))
+- Add data source and support for automatic download ([#206](https://github.com/alibaba/EasyCV/pull/206) [#229](https://github.com/alibaba/EasyCV/pull/229) [data_hub](https://github.com/alibaba/EasyCV/blob/master/docs/source/data_hub.md))
```

Comment:
Add more data source apis for open source datasets, involving classification, detection, segmentation and keypoints tasks. And part of the data source apis support automatic download. For more information, please refer to [data_hub](https://github.com/alibaba/EasyCV/blob/master/docs/source/data_hub.md) ([#206](https://github.com/alibaba/EasyCV/pull/206) [#229](https://github.com/alibaba/EasyCV/pull/229))